uk.co.compucorp.civicrm.tasksassignments/js/test/controllers/modal/modal-document.spec.js

```diff
@@ -318,6 +318,122 @@ define([
           expect(DocumentService.save).toHaveBeenCalledWith(jasmine.objectContaining({ activity_date_time: '' }));
         });
       });
+
+      describe('document contains files', function () {
```

Comment:
1. `document contains files` should be `document has attachments in upload queue` 
2. `document already has attachments` should be `document has existing attachments`
I will update it in test cases.